source/includes/steps-create-system-dsn-linux.yaml

```diff
@@ -26,31 +26,117 @@ content: |
 
```

Comment:
Your fact include may make the info about the drivers above obsolete. Have a look between them.

Reply:
Any feedback here?

Reply:
I took the filenames out of the include file. The include file used to be separated from the paragraph above by the table, but now that they're next to each other it's more clear which file is which.